test/e2e/storage/pd.go

```diff
@@ -449,7 +449,7 @@ var _ = SIGDescribe("Pod Disks", func() {
 		// Verify that disk shows up in node 0's volumeInUse list
 		framework.ExpectNoError(waitForPDInVolumesInUse(nodeClient, diskName, host0Name, nodeStatusTimeout, true /* should exist*/))
 
-		output, err := exec.Command("gcloud", "compute", "instances", "list").CombinedOutput()
+		output, err := exec.Command("gcloud", "compute", "instances", "list", "--project="+framework.TestContext.CloudConfig.ProjectID).CombinedOutput()
```

Comment:
We're using `gcloud` for a lot of networking tests too. Anyone know why we chose to run this way? What do we gain by tihs?